main.go

```diff
@@ -21,8 +23,7 @@ import (
 )
 
 func main() {
-	log.Println("Starting server on port " + os.Getenv("PORT") + "...")
-
+	log.Println("Main.go: Starting server on port " + os.Getenv("PORT") + "...")
```

Comment:
This was a mistake, I think I put that in there temporarily for debugging and forgot to take it out.